renovate.json5

```diff
@@ -1,10 +1,51 @@
 {
   $schema: "https://docs.renovatebot.com/renovate-schema.json",
   extends: [
-    "config:base",
+    "config:base"
   ],
-  ignorePaths: [
-      "src/jvmTest/resources/",
+  enabled: true,
+  enabledManagers: [
+    "gradle",
+    "gradle-wrapper",
+    "github-actions",
+  ],
+  // Will auto-merge directly, without a PR, if tests pass - else, makes a PR.
+  // Must add Renovate to 'Allow specified actors to bypass required pull requests'
```

Comment:
Ack, do you have sufficient perms to adjust it yourself, or I need to do it?

Reply:
Nah, it's fine. TBH the 'auto-merge branch' config should probably be removed. I created this config a while ago and nowadays I think it's better to be explicit and have PRs.